docs/walkthroughs/custom-integrating-with-bower-example.md

```diff
@@ -0,0 +1,130 @@
+# Custom Integration using `fossa-deps`
+
+With `fossa-deps.{yml, json, yaml}` file, FOSSA CLI can be integrated to support any package manager or custom and non-standard management solution, that is yet to be supported natively by FOSSA CLI. With the fossa-deps file, we can:
+
+- [Include manual dependencies](./../features/manual-dependencies.md)
+- [Include vendored dependencies](./../features/vendored-dependencies.md)
+
+## Example with Bower
+
+For an example, we will look at [Bower](https://bower.io/). Although, Bower team advises now, that users should migrate to npm, or yarn for new javascript based projects. We may have legacy or active projects, which are using bower. And we would like to include them in our reporting.
```

Comment:
Does adding the information about what the Bower team advises benefit this walkthrough? I think you should be able to remove this part. If you really want to keep it I would make it a note saying something like this.

> Note: Fossa does not support Bower natively. The Bower team recommends that anyone using Bower should migrate to npm or yarn for javascript projects.